homeassistant/components/light/flux_led.py

```diff
@@ -146,6 +148,8 @@ def __init__(self, device):
         self._mode = device[ATTR_MODE]
         self._bulb = None
         self._error_reported = False
+        self._color = (255, 255, 255)
```

Comment:
This looks like an invalid default now that it is HSV. Maybe `(0, 0, 100)` is better?